core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -1,4 +1,36 @@
 <script type="text/ng-template" id="skins/Conversation">
+  <!--
+    Off-screen preview of the next card in order to pre-determine the target
+    height for the card content transition animation.
+  -->
+  <div class="conversation-skin-future-tutor-card" aria-hidden="true">
```

Comment:
Yeah, at first I thought exactly the same :).

The issue was with passing a lot of variables to tutor-card, instead of using existing services, i.e PlayerTranscripsionService, OppiaPlayerService, etc...  
Furthermore, at this phase we only know a few information about upcoming card, so seems like tutor card directive feels like too much for just calculating the future height.

On the other hand, if we use the same implementation in both places the expected height will be  always the same. 

Anyway, if we go with reusing tutor-card, the code needs to be refactored, either by passing a lot of variables into the tutor-card, or by adding upcoming card into some service. If we do latter, probably need to be careful as we've some invariant about the last card. So, it deserves to be in a different CL.